.circleci/config.yml

```diff
@@ -111,6 +111,20 @@ workflows:
             - nomis-user-roles-api-prod
           requires:
             - request-prod-approval
+      - hmpps/deploy_env:
+          name: deploy_prod-cp
+          env: "prod-cp"
+          jira_update: false
```

Comment:
jira_update ,slack_notification are false here

Reply:
not sure need it enabling for both prod environments though

Reply:
that is already updated with the deployment to azure - when the full swap is done will set to true